doc/sluggable.md

```diff
@@ -94,38 +94,51 @@ on how to setup and use the extensions in most optimized way.
 
 <a name="entity-mapping"></a>
 
-## Sluggable Entity example:
+## Sluggable mapping:
 
 ### Sluggable annotations:
 
 - **@Gedmo\Mapping\Annotation\Slug** it will use this column to store **slug** generated
 **fields** option must be specified, an array of field names to slug
 
+### Sluggable attributes:
+
+- **\#[Gedmo\Mapping\Annotation\Slug]** it will use this column to store **slug** generated
```

Comment:
I know this text is copied from the previous item, but it doesn't seem reasonable to me. Is there a missing full stop after "generated"? Otherwise, I don't get the message in this sentence.